### PR TITLE
Feature/wat 37 php backend clear out

### PR DESCRIPTION
**Pull request by summarek**
Delete php files, create express + typescript template, create mysql connection config fille

### CODE REVIEW

**Comment by tomasz-jankowski** [comment]
I suggest to replace "var" keywords with "const" (or "let" if applies).

File: backend/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "watercool-backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.ts",
+  "scripts": {
+    "test": "npm run test",
+    "start": "tsc && node server.js"
```
Comment (jowczar) [inline backend/package.json]: let's add package `nodemon` (only for development) & a script called "dev" that will run it
it automatically reruns node server onchange of any files

Comment (jowczar) [inline backend/package.json]: we can drop the tsc here for sure (I have seen typescript being used directly, without compiling to js first, for development purposes)
googling a bit - maybe ts-node would be useful for this

and then add script "build" that actually builds .ts into .js files 
thanks to that, we will get rid of these generated .js files in repository

**Review by jowczar — APPROVED** [review]
GJ

**Pull request merged**